Security/Risk_management/Rapid_Risk_Assessment.mediawiki

```diff
@@ -0,0 +1,356 @@
+
+* Can this project use systems which are easier to use but provide a lower security level?
+* How much should we care about maintenance, etc?
+* Should we create a full threat-model, pen-test the service, is privacy or legal team involvement necessary?
+* How long should we spend on these tasks?
+* Is there anything obvious I should really look at fixing right now?
+
+{| class="wikitable"
+|-
+! <span style="color:gray;">Sample use case</span>
+|-
+|
+Firefox accounts store user data:
+
+* What happens if that data is exposed to the world? What happens if the systems go down? What happens if that data is disclosed? What if the data is modified?
+* Are we at financial, operational risk? Is our reputation also at risk?
+* What kind of data does Firefox accounts exactly handle? How sensitive is the primary kind of data used by Firefox accounts?
+
+The RRA table has answers to all of these questions.
+|}
+
+== When to run RRAs? ==
+
+RRAs are designed to be updated and re-runnable as needed. That being said, it is recommended to run the first RRA
+during the design or architecture phase of new services, when an initial data flow diagram for the service has been created.
+
+== What to run through RRAs? ==
+
+RRAs can only be used for services. Any project belongs to or provides a service. Any component, piece of code or data
+belongs to a service. Find which service it belongs to and RRA that service, not the specific piece of code or data.
+
+For large services, these can be split into multiple smaller services or sub-services. A choice has to be made by the
+security engineer running the RRA. If the sub-services belong to different teams, it is a strong indicator that
+multiple RRAs should be ran.
+
+== What to focus on during RRAs? ==
+
+* '''Impact assessment'''. The RRA is the authority for impact levels and these are paramount.
+* '''Rationales'''. Anyone should understand why you've set a specific impact level by reading 
+* Getting '''value for the team'''. The team needs to understand what is most important to protect.
+
+What to '''NOT''' focus on:
+* Gathering security controls, figuring out how effective they are, etc. Don't do that! This information may be recorded if it comes up but do not focus on it.
+* Likelihood, Security provided by service. Don't spent much time there! These are "10, 000 feet approximations" and part of a larger risk calculation mechanism.
+
+== Guided process: Running your RRA in ~30 minutes ==
+
+This is a guided example of how to run an RRA.
+
+{| class="wikitable"
+|-
+! <span style="color:gray;">Time management and control</span>
+|-
+|
+You will be responsible for the time distribution and ensure 30 minutes are not exceeded if possible.
+
+This means that you will sometimes have to cut a discussion short. An incomplete RRA due to time being exceeded has no
+value. Do not hesistate to take control and enforce your own discussion topics. Examples of when you'll need to be
+assertive and regain control:
```

Comment:
I'd rephrase this paragraph: "People have a tendency to jump directly to discussing security controls during risk discussions. While valuable, this is not the purpose of a RRA. You **must** maintain focus on discussion impacts and leave any discussion about control to later. Once impacts are clearly defined, controls are much easier to discuss."